pkgs/applications/audio/gtkcord4/default.nix

```diff
@@ -0,0 +1,49 @@
+{ buildGoModule
+, cairo
+, fetchFromGitHub
+, gdk-pixbuf
+, glib
+, gobject-introspection
+, graphene
+, gtk4
+, lib
+, pango
+, pkg-config
+}:
+
+buildGoModule rec {
+  pname = "gtkcord4";
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-AjHqKIW3pfc6AGDUMBbQ8FUK8ciUxngM45gutuSeTNM=";
+  };
+
+  postPatch = ''
+    rm internal/gtkcord/window/about/about.go
+  '';
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    cairo
+    gdk-pixbuf
+    glib
+    gobject-introspection
```

Comment:
You are sure about this? Normally gobject-introspection goes into nativeBuildInputs.

Reply:
I am more than happy to move it, but it does compile like this; maybe go only links against the shared library and does not call the binary?